framework/console/ErrorHandler.php

```diff
@@ -55,7 +56,9 @@ protected function renderException($exception)
             if ($exception instanceof \yii\db\Exception && !empty($exception->errorInfo)) {
                 $message .= "\n" . $this->formatMessage("Error Info:\n", [Console::BOLD]) . print_r($exception->errorInfo, true);
             }
-            $message .= "\n" . $this->formatMessage("Stack trace:\n", [Console::BOLD]) . $exception->getTraceAsString();
+            if ($previous === null) {
+                $message .= "\n" . $this->formatMessage("Stack trace:\n", [Console::BOLD]) . $exception->getTraceAsString();
```

Comment:
@samdark There's a little difference to the web error handler. It renders the call stack for the top/outer most exception. Whereas here we render the stack trace for the inner most (originating) exception. Actually I find this more helpful. The stack trace usually includes the outer methods in its trace anyway.